src/modules/fundingManager/bondingCurveFundingManager/ToposBondingCurveFundingManager.sol

```diff
@@ -132,13 +134,21 @@ contract ToposBondingCurveFundingManager is
         liquidityPool = ILiquidityPool(_liquidityPool);
         // Set formula contract
         formula = IToposFormula(bondingCurveProperties.formula);
+        // Set capital required
         _setCapitalRequired(bondingCurveProperties.capitalRequired);
+        // Set current seize to MAX_SEIZE
```

Comment:
Isn't this set to `bondingCurveProperties.seize` instead of `MAX_SEIZE`?

Reply:
You're right. This is still an old comment. Changed it